js2-mode.el

```diff
@@ -7264,7 +7264,7 @@ are ignored."
            when (and (eq 'block (js2-comment-node-format node))
                      (save-excursion
                        (goto-char (js2-node-abs-pos node))
-                       (looking-at "/\\*global ")))
+                       (looking-at "/\\*[ ]?global ")))
```

Comment:
Eslint does work with multiple spaces.  I think `" +` is better because characters other than space probably shouldn't be accepted.  `/* x global foobar */` shouldn't work, in my opinion.